app/views/candidate_interface/degrees/degree/new_subject.html.erb

```diff
@@ -6,8 +6,16 @@
   <%= form_with model: @wizard, url: candidate_interface_new_degree_subject_path do |f| %>
     <%= f.govuk_error_summary %>
     <%= f.govuk_fieldset legend: { text: 'What subject is your degree?', size: 'l' } do %>
-      <%= f.govuk_text_field :subject, hint: { text: t('application_form.degree.subject.hint') }, label: -> {} %>
-      <%= tag.div(id: 'degree-subject-autosuggest', data: { source: @wizard.subjects }) %>
+      <%= render DfE::Autocomplete::View.new(
+        f,
+        attribute_name: :subject,
+        form_field: f.govuk_select(
+        :subject,
+        options_for_select(dfe_autocomplete_options(@wizard.subjects), f.object.subject),
+        label: { text: t('application_form.degree.subject.label'), size: 'xl', tag: 'h1' },
```

Comment:
theres a legend on the fieldset

Reply:
Resolved. I used the label as [can be one or the other.](https://design-system.service.gov.uk/get-started/labels-legends-headings)